src/Groups/sub.jl

```diff
@@ -248,21 +254,26 @@ end
 """
     issolvable(G::GAPGroup)
 
-Return whether the `G` is solvable.
+Return whether `G` is solvable,
+i. e., whether [`derived_series`](@ref)(`G`)
```

Comment:
The simpler following version should also work:
```suggestion
i. e., whether [`derived_series(G)`](@ref)
```

Reply:
@rfourquet Yes, this works, thanks. However, I prefer the "more involved" variant because it creates a hyperlink on "derived_series", whereas your variant creates a hyperlink on "derived_series(G)".